Source/Scene/Scene.js

```diff
@@ -1684,6 +1586,24 @@ define([
     function updatePrimitives(scene) {
         var frameState = scene._frameState;
 
+        // Update celestial and terrestrial environment effects.
+        var renderPass = frameState.passes.render;
```

Comment:
What do you think of moving this to a separate top-level function `updateEnvironment` like `updatePrimitives`?